src/vulkan/vulkan_renderer.cpp

```diff
@@ -73,6 +73,8 @@ namespace IC {
     }
 
     void VulkanRenderer::DrawFrame() {
+        auto start = std::chrono::system_clock::now();
```

Comment:
`std::chrono::high_resolution_clock` would be more accurate here. I think on Windows `system_clock` only has millisecond precision.

Reply:
GLFW also has `glfwGetTime()`, that might be better for a cross-platform approach. Not sure on the precision on it.